pkg/handler/pullrequest/handler.go

```diff
@@ -65,17 +66,36 @@ func logPullRequestInfo(pr *github.PullRequestPayload) {
 func openOrSync(gitRepo *git.Git, pr *github.PullRequestPayload, gh ghclient.GH) error {
 	prNum := int(pr.Number)
 
+	config, err := repoconfig.Read(&repoconfig.Source{
+		SSHURL: pr.PullRequest.Base.Repo.SSHURL,
+		RepoName: pr.PullRequest.Base.Repo.FullName,
+		SHA: pr.PullRequest.Base.Sha,
+		Remote: "origin",
+	})
+
+	if err != nil {
+		klog.Errorf("reading bot config: %s", err)
+		return err
+	}
```

Comment:
This will cause the handler to exit and not handle the rest..

I suggest in this case you just ignore the parts relating to the label but still do the branch handling

Reply:
Done